apps/analytics/urls.py

```diff
@@ -10,5 +10,7 @@
         views.get_participant_count, name='get_participant_count'),
     url(r'challenge/(?P<challenge_pk>[0-9]+)/submission/(?P<duration>[A-Za-z]+)/count$',
         views.get_submission_count, name='get_submission_count'),
-
+    url(r'challenge/(?P<challenge_pk>[0-9]+)/challenge_phase/'
+        r'(?P<challenge_phase_pk>[0-9]+)/last_submission/(?P<submission_by>[A-Za-z_]+)$',
```

Comment:
@deshraj I think `submission_by` is correct here as I am returning the time for who is asking for the submission time. Please tell if I am not clear.